projects/TicTacToeTkInter/tic-toc-toe-tkinter.py

```diff
@@ -0,0 +1,158 @@
+        b3.config(bg="red")
+        b10["text"] = f"{whoseTurn()} Won"
+        disable_all_button()
+
+    elif(b4["text"] != " " and (b4["text"] == b5["text"] == b6["text"])):
+        b4.config(bg="red")
+        b5.config(bg="red")
+        b6.config(bg="red")
+        b10["text"] = f"{whoseTurn()} Won"
+        disable_all_button()
+
+    elif(b7["text"] != " " and (b7["text"] == b8["text"] == b9["text"])):
+        b7.config(bg="red")
+        b8.config(bg="red")
+        b9.config(bg="red")
+        b10["text"] = f"{whoseTurn()} Won"
+        disable_all_button()
+
+    elif(b1["text"] != " " and (b1["text"] == b5["text"] == b9["text"])):
+        b1.config(bg="red")
+        b5.config(bg="red")
+        b9.config(bg="red")
+        b10["text"] = f"{whoseTurn()} Won"
+        disable_all_button()
+
+    elif(b3["text"] != " " and (b3["text"] == b5["text"] == b7["text"])):
+        b3.config(bg="red")
+        b5.config(bg="red")
+        b7.config(bg="red")
+        b10["text"] = f"{whoseTurn()} Won"
+        disable_all_button()
+
+    elif (count == 9):
+        b10["text"] = "Its tie, No one Won!!"
+        disable_all_button()
+
+
+def b_click(b):
+    global clicked, count, b10
+
+    if b["text"] == " ":
+        b["text"] = whoseTurn()
+        count += 1
+        b10["text"] = "X Turn" if clicked is False else "O Turn"
+        is_win()
+        clicked = not clicked
+
+
+def reset():
+    global b1, b2, b3, b4, b5, b6, b7, b8, b9, b10
+    global clicked, count
+
+    clicked = True
+    count = 0
+
+    b1 = Button(root, text=" ", font=("Helvetica,20"), height=3,
+                width=6, bg="SystemButtonFace", command=lambda: b_click(b1))
+    b2 = Button(root, text=" ", font=("Helvetica,20"), height=3,
+                width=6, bg="SystemButtonFace", command=lambda: b_click(b2))
+    b3 = Button(root, text=" ", font=("Helvetica,20"), height=3,
+                width=6, bg="SystemButtonFace", command=lambda: b_click(b3))
+
+    b4 = Button(root, text=" ", font=("Helvetica,20"), height=3,
+                width=6, bg="SystemButtonFace", command=lambda: b_click(b4))
+    b5 = Button(root, text=" ", font=("Helvetica,20"), height=3,
+                width=6, bg="SystemButtonFace", command=lambda: b_click(b5))
+    b6 = Button(root, text=" ", font=("Helvetica,20"), height=3,
+                width=6, bg="SystemButtonFace", command=lambda: b_click(b6))
+
+    b7 = Button(root, text=" ", font=("Helvetica,20"), height=3,
+                width=6, bg="SystemButtonFace", command=lambda: b_click(b7))
+    b8 = Button(root, text=" ", font=("Helvetica,20"), height=3,
+                width=6, bg="SystemButtonFace", command=lambda: b_click(b8))
+    b9 = Button(root, text=" ", font=("Helvetica,20"), height=3,
+                width=6, bg="SystemButtonFace", command=lambda: b_click(b9))
+
+    b10 = Button(root, text=f'{whoseTurn()} Turn', font=(
+        "Helvetica,20"), height=2, width=20, bg="Black", fg="white")
```

Comment:
make it modular by doing all manipulation inside functions and call them with some parameter